x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning.tsx

```diff
@@ -34,40 +40,195 @@ interface Props {
 }
 
 export const RelevanceTuning: React.FC<Props> = ({ engineBreadcrumb }) => {
-  const { initializeRelevanceTuning } = useActions(RelevanceTuningLogic);
+  const { resetSearchSettings, initializeRelevanceTuning, updateSearchSettings } = useActions(
+    RelevanceTuningLogic
+  );
+  const { schemaFieldsWithConflicts, engineHasSchemaFields, unsavedChanges } = useValues(
+    RelevanceTuningLogic
+  );
+  const {
+    engine: { invalidBoosts, unsearchedUnconfirmedFields },
+  } = useValues(EngineLogic);
+
+  const schemaFieldsWithConflictsCount = schemaFieldsWithConflicts.length;
 
   useEffect(() => {
     initializeRelevanceTuning();
   }, []);
 
   return (
     <>
+      <UnsavedChangesPrompt hasUnsavedChanges={unsavedChanges} />
       <SetPageChrome trail={[...engineBreadcrumb, RELEVANCE_TUNING_TITLE]} />
-      <EuiPageHeader>
-        <EuiPageHeaderSection>
-          <EuiTitle size="l">
-            <h1>{RELEVANCE_TUNING_TITLE}</h1>
-          </EuiTitle>
-          <EuiText>
-            <EuiTextColor color="subdued">
+      <EuiPageHeader
+        className="relevanceTuningHeader"
+        pageTitle={RELEVANCE_TUNING_TITLE}
+        description={i18n.translate(
```

Comment:
Hmm just noticed EuiPageHeader is adding a forced `EuiSpacer` between the pageTitle and description that we can't seem to customize :( Unfortunate, there's some odd-looking spacing on this page as a result

Reply:
I bugged the EUI team but it looks like the EuiSpacer thing is meant to be 😛 Collapsing this comment for now, I might do a final negative space pass with Davey at some point whenever you think the UI is almost done / ready for that Jason